Application/Features/UserFeature/Queries/GetUserByIdQuery.cs

```diff
@@ -29,7 +30,13 @@ public GetUserByIdQueryHandler(IMapper mapper, UserManager<User> userManager)
         }
         public async Task<GetUserByIdQueryResponse> Handle(GetUserByIdQueryRequest request, CancellationToken cancellationToken)
         {
-            var appUser = await _userManager.Users.FirstOrDefaultAsync(u => u.Id == request.Id, cancellationToken);
+            User appUser = null;
+            if (request.Id != null)
+                appUser = await _userManager.Users.FirstOrDefaultAsync(u => u.Id == request.Id, cancellationToken);
```

Comment:
if we provide both params we will call db twice

Reply:
`TelegramId` will be used by our Bot, the `Id` will be the frontend application or everything else? I mean we will never use both of them, if both are used then `telegramId` will have higher priority (since it's also unique).

Another option: I can request a list of all users, and select one which the same `telegramId `as our current user.